src/services/API/Plio.js

```diff
@@ -115,6 +116,15 @@ export default {
     return apiClient().delete(pliosEndpoint + plioId);
   },
 
+  /**
+   * Copy a given plio to another workspace
+   * @param {Number} plioId - uuid id of the plio
+   * @param {Object} payload - params required for copying the plio
+   */
+  copy(plioId, payload) {
```

Comment:
Should we rename this?
Copy sounds very generic. What we have is very specific, copying to a workspace

Reply:
copyToWorkspace